addons/event/migrations/12.0.1.0/post-migration.py

```diff
@@ -1,18 +1,21 @@
 # Copyright 2018 Eficent <http://www.eficent.com>
 # Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Tecnativa - Jairo Llopis
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openupgradelib import openupgrade
+from openupgradelib import openupgrade, openupgrade_120
 
 
-@openupgrade.migrate(use_env=False)
```

Comment:
@mreficent look why I always want to have `env` by default...

Reply:
yeah, from some time ago, I do always do `env` by default...